### PR TITLE
feat(helm): add labels to volumesnapshotclass

### PR DESCRIPTION
**Pull request by fad3t**
Hello,

This PR is adding the option to set labels to the volume snapshot classes created by the Helm chart.
For now this is only possible on the storage classes.

Thanks,
Fred

### CODE REVIEW

**Comment by AndrewSirenko** [comment]
Thanks for your contribution! (And great profile icon) 
/ok-to-test

File: charts/aws-ebs-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.23.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.23.0
+version: 2.24.0
```
Comment (ConnorJC3) [inline charts/aws-ebs-csi-driver/Chart.yaml:5]: Please remove this diff from your PR. We generally do not accept PRs that add a feature and perform a release at the same time - releases are performed by our maintainers on a semi-regular schedule and take into account details such as backwards compatibility, our support policy, updating the changelog, etc.

Other than that this PR lgtm.

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]

**Comment by fad3t** [comment]
Thanks for the reviews! I've removed the version bump.

**Pull request merged**